scheduler/src/cook/kubernetes/api.clj

```diff
@@ -977,14 +977,30 @@
   (when (and mode volume-name)
     (make-empty-volume volume-name)))
 
+(defn resolve-incremental-config-volume-mounts
```

Comment:
Why the choice of 'checkpoint' for the variable name?

Can you explain a bit of the why this exists. Not what it does. but why you might want to do this? (E.g., migration to a new image? Or is there something else?)

Reply:
Then can you add that to the comment? The code has 'volume' in it, your response is that its involved with checkpointing and incremental rollout. Can you explain that gap in a comment?

Reply:
updated